Formula/bcc.rb

```diff
@@ -0,0 +1,32 @@
+class Bcc < Formula
+  desc "Tools for BPF-based Linux IO analysis, networking, monitoring, and more"
+  homepage "https://github.com/iovisor/bcc"
+  url "https://github.com/iovisor/bcc/releases/download/v0.24.0/bcc-src-with-submodule.tar.gz"
```

Comment:
No worries. Do you have an example of this?

Reply:
I'm not certain, but I'm afraid the presence of the submodule code is tightly coupled with the build process.

Reply:
It seems like it could make sense to follow suite with what Debian does here: https://packages.debian.org/source/sid/bpfcc, although I'm not interested in all of these at this time.